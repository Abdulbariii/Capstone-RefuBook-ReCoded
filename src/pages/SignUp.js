import React from 'react';
import SignUpPage from '../components/signUp-signIn/SignUpPage';


export default function SignUp() {
  return <div>
    <SignUpPage />
  </div>;
}
