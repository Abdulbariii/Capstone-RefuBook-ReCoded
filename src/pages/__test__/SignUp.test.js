import React from 'react';
import { render } from '@testing-library/react';
import SignUp from '../SignUp';
import { BrowserRouter } from "react-router-dom"

const MockSignUp = () => {
    return (
        <BrowserRouter>
            <SignUp />
        </BrowserRouter>
    )
}
test('matches snapshot', () => {
    const { asFragment } = render(<MockSignUp />);
    expect(asFragment()).toMatchSnapshot();
});
