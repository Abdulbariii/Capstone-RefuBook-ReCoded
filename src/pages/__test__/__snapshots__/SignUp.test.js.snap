// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="min-h-screen"
  >
    <section
      class="flex flex-col min-h-screen"
    >
      <div
        class="fixed z-[-10]  lg:top-0 left-0 bottom-0 lg:right-auto right-0 lg:w-[50%] w-full lg:h-full h-[40%] bg-[#e5faff]  rounded-t-[50%]  lg:rounded-l-[24px]  lg:rounded-r-full  "
      />
      <div
        class="z-20 lg:m-auto lg:p-9 lg:mt-48 mt-24 mx-10 p-16 flex flex-col justify-between items-center lg:gap-6 text-center bg-white shadow-[2px_4px_8px_rgba(0,0,0,0.3)] rounded-3xl "
      >
        <p
          class="text-[#4699C2] font-sans font-bold lg:text-2xl text-lg text-center"
        >
          SIGN UP WITH
        </p>
        <div
          class="flex lg:flex-row flex-col justify-between items-center lg:gap-10 "
        >
          <button
            class="px-16 py-2   mt-5 bg-[#eb5757] text-white font-bold lg:text-lg text-base rounded-full shadow-md hover:bg-[#df3535] hover:shadow-lg focus:bg-[#df3535] focus:shadow-lg focus:outline-none focus:ring-0 active:bg-[#df3535] active:shadow-lg transition duration-150 ease-in-out"
            type="button"
          >
            <img
              alt="Google logo"
              class="w-8 h-8"
              src="Google.png"
            />
          </button>
          <p
            class="text-[#4699C2] font-sans font-bold text-lg text-center mt-4"
          >
            OR
          </p>
          <button
            class="px-16 py-2   mt-5 bg-[#2F80ED] text-white font-bold lg:text-lg text-base rounded-full shadow-md hover:bg-[#026FC2] hover:shadow-lg focus:bg-[#026FC2] focus:shadow-lg focus:outline-none focus:ring-0 active:bg-[#026FC2] active:shadow-lg transition duration-150 ease-in-out"
            type="button"
          >
            <img
              alt="facebook logo"
              class="w-8 h-8"
              src="Facebook.png"
            />
          </button>
        </div>
        <div
          class="lg:static absolute bottom-[15%] flex lg:flex-row flex-col justify-center items-center lg:gap-0 gap-2"
        >
          <p
            class="lg:text-[#a5e7fa] text-[#70CDD6] font-sans font-bold text-lg text-center"
          >
            Already a member?
          </p>
          <a
            class=""
            href="/login"
          >
            <button
              class=" lg:bg-transparent bg-[#2F80ED] lg:text-[#70CDD6] text-white lg:p-2  px-8 py-1 hover:lg:font-black  lg:rounded-none rounded-full  hover:bg-[#026FC2] hover:lg:bg-transparent  focus:bg-[#026FC2] focus:lg:bg-transparent focus:outline-none font-bold text-lg text-center"
              type="button"
            >
              Sign in
            </button>
          </a>
        </div>
      </div>
       
    </section>
  </div>
</DocumentFragment>
`;
